plugins/woocommerce/client/admin/client/wp-admin-scripts/print-shipping-label-banner/shipping-banner/test/index.js

```diff
@@ -643,7 +660,7 @@ describe( 'If incompatible WCS&T is active', () => {
 			expect( acceptWcsTos ).not.toHaveBeenCalled();
 		} );
 
-		const notice = getByText( ( _, element ) => {
+		const notice = await findByText( ( _, element ) => {
```

Comment:
`findByText` works because it waits for the element to appear in the DOM (asynchronous).